src/types/externalRouteProps.ts

```diff
@@ -1,9 +1,10 @@
+import { Host } from '@/types/host'
 import { Path } from '@/types/path'
 import { Query } from '@/types/query'
 import { Route } from '@/types/route'
 
 // eslint-disable-next-line @typescript-eslint/no-unnecessary-type-arguments
-export type ExternalRoutes = Route<string, '', Path, Query, boolean>[]
+export type ExternalRoutes = Route<string, Host, Path, Query, boolean>[]
```

Comment:
I believe this should be 
```suggestion
export type ExternalRoutes = Route<string, Host<''>, Path, Query, boolean>[]
```

It's restricting children in external routes from having a host. This type should really be called `ExternalChildren` or something.

Reply:
solved this together